### ISSUE TITLE
non-yaml files claimed to be correct

### ISSUE THREAD

**Issue by pputnik** [adrienverge/yamllint#256]
When I test just random text file the only message I get is warning and exit code=0
```
$ yamllint tmp.txt
tmp.txt
  1:1       warning  missing document start "---"  (document-start)
$ echo $?
0
```

**Comment by adrienverge** [comment]
Can you please give a reproductible case, and more info on what you expect? It's hard to understand your problem with just one sentence.

**Comment by pputnik** [comment]
Sure.
Fresh setup, so no configs:
```

cat tmp.txt
lkjlkjkj
XDgvzdfgbnxfgh

yamllint tmp.txt
tmp.txt
  1:1       warning  missing document start "---"  (document-start)

vim .yamllint
---

extends: default
rules:
  document-start: disable
:wq

yamllint tmp.txt
[nothing here]
echo $?
0

```

**Comment by adrienverge** [comment]
```
lkjlkjkj
XDgvzdfgbnxfgh
```
**is** valid YAML, it's a representation of the string `"lkjlkjkj XDgvzdfgbnxfgh"`.

Can you try changing extension from `tmp.txt` to `tmp.yaml` and load it with your favorite YAML loader?

**Comment by pputnik** [comment]
My apologies.
